package.json

```diff
@@ -88,14 +90,12 @@
       "jest-extended"
     ]
   },
-  "husky": {
```

Comment:
Must have: This PR will remove commitlint. We don't want to remove this. Let's make sure both lint-staged and commitlint are still running.

Reply:
Done.